emails.coffee

```diff
@@ -57,9 +57,15 @@ Helpers =
   # creates an absolute URL.
   #
   #     {{ emailUrlFor 'myRoute' param='foo' }} => http://root-domain.com/my-route/foo
-  emailUrlFor: (route, params) ->
-    if Router
-      Utils.joinUrl Mailer.settings.baseUrl, Router.path.call(Router, route, params.hash)
+  emailUrlFor: (routeName, params) ->
+    # if Iron Router
+    if Router?
+      Utils.joinUrl Mailer.settings.baseUrl, Router.path.call(Router, routeName, params.hash)
+
+    # if Flow Router
+    if FlowRouter?
+      baseUrl = Utils.joinUrl Mailer.settings.baseUrl, FlowRouter.path.call(FlowRouter, routeName, params.hash)
```

Comment:
Did you mean to store this in `baseUrl` here?

I also think we should have a fallback mechanism if the user has neither FlowRouter nor Iron Router installed. Like, doing a logger warning in the console.